source/features/view-markdown-source.tsx

```diff
@@ -50,6 +50,10 @@ This acts as an auto-discarded cache without globals, timers, etc.
 It should also work clicks on buttons sooner than the page loads.
 */
 async function showSource(): Promise<void> {
+	const newURL = window.location.protocol + "//" + window.location.host + window.location.pathname + '?plain=1' + window.location.hash;
```

Comment:
It would be better to use `URL` or URL manipulation.

Reply:
> It would be better to use `URL` or URL manipulation.

So is it okay to add it to the dependencies? 🤔

Reply:
> It would be better to use `URL` or URL manipulation.

On it! :)